examples/general/abs_home_path/src/lib.rs

```diff
@@ -227,3 +173,69 @@ fn build_script_allowance() {
         "The abs_home_path lint was incorrectly triggered in a build script: {stderr}"
     );
 }
+
+#[test]
+fn test_context_allowance() {
```

Comment:
This test is very similar to `build_script_allowance`.

Could you combine the two so that, in the end, there is one test with a two-iteration loop?

Reply:
Something I forgot to mention: the data structure that you loop over could include a closure to perform the checks after `cargo-dylint` is run.

In this way, the tests could perform different checks.